pandas/tests/io/parser/test_network.py

```diff
@@ -31,24 +31,18 @@ def setup(self):
                          'pandas/io/tests/parser/data/salaries.csv')
```

Comment:
This test might be broken; I don't see this file, with or without the compressed extension.